internal/images/image.go

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package images
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/eclipse/che-plugin-broker/model"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("container-images")
+
+var envRegexp = regexp.MustCompile(`\${(.*)}`)
```

Comment:
I think it makes sense to introduce convention that the only `IMAGE_*` env var could be filled in in plugin configuration, otherwise, every workspace creator could get any env var from controller with custom plugin configuration. WDYT?

Reply:
How secret are env vars supposed to be here? Either way I'll add it to the regex.

Reply:
> How secret are env vars supposed to be here?

I know this is not really suitable example but still https://github.com/eclipse/che/blob/master/deploy/openshift/templates/multi/postgres-template.yaml#L44
Who knows all env vars K8s infrastructure injects? And how we make sure that like SA token is not injected with env var some day?

Reply:
Fair enough :)